python/BioSimSpace/Process/_amber.py

```diff
@@ -55,7 +63,9 @@
 from BioSimSpace import _Utils as _Utils
 
 from . import _process
-from ._plumed import Plumed as _Plumed
+
+_Plumed = _try_import("Plumed")
+
```

Comment:
Plumed is currently an optional external dependeny, like all of the molecular dynamics engines. You shouldn't need to add this as a conda dependency and check that it's installed. As with the other packages, we use `findExe` to locate the `plumed` executable within the users `PATH`. I'm happy to add plumed` as a conda dependency if it's easier.